src/os/filestore/FileStore.cc

```diff
@@ -718,6 +718,14 @@ int FileStore::statfs(struct store_statfs_t *buf0)
   }
   buf0->total = buf.f_blocks * buf.f_bsize;
   buf0->available = buf.f_bavail * buf.f_bsize;
+  // Adjust for writes pending in the journal
+  if (journal) {
+    uint64_t estimate = journal->get_journal_size_estimate();
```

Comment:
@tchaikov thanks for reply.
Let's see your second case. As I understand:
```
data.used = data flushed in disk  + data in page cache.
journal.used = spaces of journal item.
```
Let's see the situation: is it possible in some time, some data both in journal.used, in data.used?